src/components/Home/Navbar/index.jsx

```diff
@@ -25,14 +34,14 @@ export function Navbar() {
                             <Link to="/wishlist"><p>Wishlist</p></Link>
                             <div className="badge num-notification">
                                 <i className="fa fa-bell"></i>
-                                <span className="badge-notification-count">7</span>
+                                <span className="badge-notification-count">{wishlist.length}</span>
                             </div>
                     </li>
                     <li>
                             <Link to="/cart"><p>Cart</p></Link>
                             <div className="badge num-cart">
                                 <i className="fa fa-shopping-cart"></i>
-                                <span className="badge-cart-count">3</span>
+                                <span className="badge-cart-count">{cart.length}</span>
```

Comment:
Maybe a font size increase on the counter of the cart and wishlist might look good as well